Makefile

```diff
@@ -66,9 +74,13 @@ install:
 build-linux:
 	# Linux images for docker-compose
 	# CGO_ENABLED=0 added to solve this issue: https://stackoverflow.com/a/36308464
-	BIN_PREFIX=-linux LEDGER_ENABLED=false GOOS=linux CGO_ENABLED=0 GOARCH=amd64 $(MAKE) build
+	BIN_PREFIX=-linux LEDGER_ENABLED=false GOOS=linux GOARCH=amd64 $(MAKE) build
+	docker run --rm --entrypoint cat emoney/cosmovisor /go/bin/cosmovisor > build/cosmovisor
+	chmod +x build/cosmovisor
+	docker run --rm --entrypoint cat emoney/test-upg /go/src/em-ledger/build/emd-linux > "build/emdupg-linux"
+	chmod +x "build/emdupg-linux"
```

Comment:
`emdupg-linux`: that's the Docker linux binary as per the sdk standand: default above, -linux for container use.
`cosmovisor` Docker use only linux binary. I didn't append the linux suffix because there isn't an OS default binary in this case.

For a general use cosmovisor binary, I have the `go install` instruction in the [Readme](https://github.com/e-money/em-ledger/blob/27-upgrade/networks/upg/README.md) file